### ISSUE TITLE
HTTP3 support

### ISSUE THREAD

**Issue by szmarczak** [sindresorhus/got#843]
Related:
- https://http3-explained.haxx.se/en/
- https://github.com/nodejs/node/issues/23064
- https://github.com/nodejs/quic

**Comment by duongvanba** [comment]
When we can use this future ?

**Comment by szmarczak** [comment]
Once it lands in Node.js.

**Comment by duongvanba** [comment]
Hi @szmarczak 
I see Quick support in nodejs v15
So we will support it now right ?

**Comment by szmarczak** [comment]
Not now. We need to wait for HTTP/2 to become stable enough so we can make it the default protocol.

**Comment by szmarczak** [comment]
There are still issues with HTTP/2. We can't default to HTTP/2 yet.

**Comment by gokaybiz** [comment]
Instead of making http2 default protocol and http3 as a side protocol, Why can't we add another option alongside http1.1 http2?
